lib/utility/_query.js

```diff
@@ -13,7 +13,7 @@ exports._query = function _query(query, data, options, callback, executor) {
 		callback = options;
 
 		// Checks if data contains options
-		if (data && data.useResultSet) {
+		if (data && (data.useResultSet || data.write)) {
```

Comment:
If more options will be added in the future it might be good to use a more general detection than explicitly checking for each possible option. Something like this:

```
if (_.isObject(data)) {
  options = data;
  data = null;
}
```